action.yaml

```diff
@@ -10,6 +10,19 @@ inputs:
     default: false
     required: false
     type: boolean
+  delete_topics:
+    description: |
+      Whether to delete topics from the documentation server if they are no
+      longer needed.
+    default: true
+    required: false
+    type: boolean
+  draft_mode:
```

Comment:
Can we rename this to ` dry-run`. The current name doesn't look obvious to me

Reply:
Yes happy to change that. Is it ok if this is done in the next PR? I'll make that change on `alpha-5` which is what the indico PR is based on. The reason is that if I make the change here, I would have to probably resolve a lot of merge commits whereas on `alpha-5` I won't have to

Reply:
This change has now been made on `alpha-5` and on the Indico documentation PR. Closing for now, please let me know if you have any further feedback.